docs: reboot CONTRIBUTING.md and PR template to reflect 4-stage manifesto (#18708)
LedgerHQ/ledger-live: docs/contributing/pr-review-guide.md

# PR Review Guide

> A practical reference for reviewing PRs in the `ledger-live` monorepo.

Reviewer: **suggestion: let's avoid the name? live/wallet**

```suggestion
A practical reference for reviewing PRs in this repository.
```
